internal/orchestrator/applications_handler.go

```diff
@@ -81,6 +83,21 @@ func (handler ApplicationsHandler) List(w http.ResponseWriter, _ *http.Request)
 	for _, rec := range records {
 		list.Applications = append(list.Applications, Application{ID: rec.ID, IntegrationId: rec.IntegrationId, ObjectId: rec.ObjectId, Name: rec.Name, Description: rec.Description, ProviderName: integrationNamesById[rec.IntegrationId], Service: rec.Service})
 	}
+
+	// sort by "Provider" so  that all app resources from a platform are grouped together.
+	sort.Slice(list.Applications, func(i, j int) bool {
+		if list.Applications[i].ProviderName < list.Applications[j].ProviderName {
```

Comment:
For better readability, please can you use [strings.Compare](https://go.dev/src/strings/compare.go) along with [swith/case](https://go.dev/tour/flowcontrol/9)
```
providerComp := strings.Compare(list.Applications[i].ProviderName, list.Applications[j].ProviderName)
serviceComp := strings.Compare(list.Applications[i].Service, list.Applications[j].Service)

switch(providerComp) {
   case 0: ... handle i == j
   case -1: .. handle i < h
   default: ... handle i > j
}
```

Reply:
Okay, let me take a look at the coderevieww you gave me. Thank you for your suggestion. I'll adjust it